CITATION.cff

```diff
@@ -3,7 +3,7 @@
 cff-version: 1.0.3
 message: If you use this software, please cite it using these metadata.
 # FIXME title as repository name might not be the best name, please make human readable
-title: 'openradar/xradar: xradar v0.3.0'
+title: 'openradar/xradar: xradar v0.3.1'
```

Comment:
I see it's necessary to change this, but your enhancement is no patch at all -> do a 0.4.0 here.